crates/viewer/re_viewer_context/src/space_view/space_view_class_placeholder.rs

```diff
@@ -53,7 +54,12 @@ impl SpaceViewClass for SpaceViewClassPlaceholder {
         _query: &ViewQuery<'_>,
         _system_output: SystemExecutionOutput,
     ) -> Result<(), SpaceViewSystemExecutionError> {
-        ui.centered_and_justified(|ui| ui.label(self.help_text(ctx.egui_ctx)));
+        ui.centered_and_justified(|ui| {
+            ui.markdown_ui(
+                egui::Id::new(self.display_name()),
+                &self.help_markdown(ctx.egui_ctx),
+            );
+        });
```

Comment:
Is this used anywhere? I wonder how well `egui_commonmark` works in a centered and justified layout

Reply:
Good call. This is only displayed when an unknown space view is spawned. I'll just remove the centring, it's not important. That's the only place we call `help_markdown()` beyond actually showing it in with the `?` buttons.